### PR TITLE
Ports Utility Outfits +

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Ports several outfits from oldbase, with the focus being Utility Uniforms - dark-grey uniforms with departmental markings, along with the same stats as their departments' traditional uniform. It also includes several costumes and other quality outfits and berets. All are available in loadouts, vendors, or both.

## Why It's Good For The Game
More outfits is nice. Style good.

## Changelog
:cl:
add: Added Utility Uniforms, available in loadouts and departmental vendors.
add: Added several purpose-made outfits, such as a gas/chemical hazard uniform in the engineering vendor.
add: Added departmental berets, available in loadouts and departmental vendors.
add: Added several new outfits based around traditional Japanese clothes and Victorian-era clothing.
/:cl:

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -3629,6 +3629,7 @@
 #include "modular_skyrat\modules\customization\modules\clothing\ears\ears.dm"
 #include "modular_skyrat\modules\customization\modules\clothing\hands\hands.dm"
 #include "modular_skyrat\modules\customization\modules\clothing\hands\rings.dm"
+#include "modular_skyrat\modules\customization\modules\clothing\hands\wrists.dm"
```
Comment (ghost) [inline tgstation.dme]: WHY IS THIS HERE I DIDNT SELECT IT

File: tgstation.dme [hunk 2]
```diff
@@ -3629,7 +3629,6 @@
 #include "modular_skyrat\modules\customization\modules\clothing\ears\ears.dm"
 #include "modular_skyrat\modules\customization\modules\clothing\hands\hands.dm"
 #include "modular_skyrat\modules\customization\modules\clothing\hands\rings.dm"
-#include "modular_skyrat\modules\customization\modules\clothing\hands\wrists.dm"
```
Comment (ghost) [inline tgstation.dme]: This is a separate project I'm doing and the fact its this desperate to get uploaded means I'll put it off for longer

**Comment by ghost** [comment]
I'll get it eventually I swear

**Comment by ghost** [comment]
God. It only took a total of like 20 hours of work. But my first PROPER PR is done and not conflicting.

**Comment by Gandalf2k15** [comment]
I like how you've centralized the icon files for the clothing. We should just have them all in masterfiles.

**Review by Gandalf2k15 — APPROVED** [review]
